analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/TestHelper.cs

```diff
@@ -59,6 +61,17 @@ public static (SyntaxTree, SemanticModel) Compile(string snippet, bool ignoreErr
             return (compiled.SyntaxTree, compiled.SemanticModel);
         }
 
+        public static ControlFlowGraph CompileCfgBodyCS(string body = null) =>
+            CompileCfg($"public class Sample {{ public void Main() {{ {body} }} }}");
+
+        public static ControlFlowGraph CompileCfgBodyVB(string body = null) =>
+            CompileCfg(
+$@"Public Class Sample
+    Public Sub Main()
+        {body}
+    End Sub
+End Class", false);
```

Comment:
nitpick (totally optional): using `isCSharp: false` named comment would increase readability (even if the file is small, I prefer verbosity in test code)

Reply:
I actually don't like the `bool isCSharp` for being `bool` in general. It should be `AnalyzerLanguage language` instead. I'll get rid of it in another PR.

Reply:
See #5132